.circleci/config.yml

```diff
@@ -156,6 +156,39 @@ commands:
           environment:
             SYNCSTORAGE_RS_IMAGE: app:build
 
+  run-spanner-tests:
+    steps:
+      - run:
+          name: cargo spanner build
+          command: cargo build --no-default-features --features=syncstorage-db/spanner --features=py_verifier
+      - run:
+          name: Wait for Spanner Emulator
+          command: |-
+            for i in {1..10}; do
+              if nc -z 127.0.0.1 9020; then
+                echo "Spanner emulator running on port 9020."
+                break
+              fi
+              echo "Waiting for Spanner emulator..."
+              sleep 2
+            done
+            if ! nc -z 127.0.0.1 9020; then
+              echo "ERROR: Cannot connect to Spanner emulator at 127.0.0.1:9020. Terminating."
+              exit 1
+            fi
+      - run:
+          name: setup spanner
+          environment:
+            SYNC_SYNCSTORAGE__DATABASE_URL: spanner://projects/test-project/instances/test-instance/databases/test-database
+            SYNC_SYNCSTORAGE__SPANNER_EMULATOR_HOST: 127.0.0.1:9020
+          command: scripts/prepare-spanner-unit-tests.sh
+      - run:
+          name: cargo test spanner
+          environment:
+            SYNC_SYNCSTORAGE__DATABASE_URL: spanner://projects/test-project/instances/test-instance/databases/test-database
+            SYNC_SYNCSTORAGE__SPANNER_EMULATOR_HOST: 127.0.0.1:9020
+          command: cargo test --no-default-features -p syncstorage-spanner
```

Comment:
I think we should `cargo test` basically the entire test suite vs just the `syncstorage-spanner` crate.

There's a number of tests that wouldn't utilize the spanner code at all (like tokenserver*) but just `syncstorage-spanner` isn't enough: our larger db unit test suite lives in the `syncstorage-db` crate, and additionally `syncserver` itself has a [integration test suite](https://github.com/mozilla-services/syncstorage-rs/blob/master/syncserver/src/server/test.rs) that utilizies the database.